app/views/users/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
```

Comment:
- [x] This activity requires the modification of the views in _devise_ for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly, like seen in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md):

**Your login view:**        |  **Wireframe login view:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/Rachelwebdev/blog-app/assets/12745474/d7f0e623-e4f0-4b8e-870c-264574c35df0"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png">